include/modem/lte_lc.rst

```diff
@@ -0,0 +1,44 @@
+.. _lte_lc_readme:
+
+LTE link controller
+###################
+
+The LTE link controller library is intended to be used for simpler control of LTE connections on an nRF9160 SiP.
+
+This library provides the API and configurations for setting up an LTE connection with definite properties.
+Some of the properties that can be configured are Access Point Name (APN), LTE mode (NB-IoT or LTE-M), Power Saving Mode (PSM), and extended Discontinuous Reception (eDRX).
```

Comment:
I do not think it is only LTE modes. What about GNSS support and LTE preferences?

```suggestion
Some of the properties that can be configured are Access Point Name (APN), support for LTE modes (NB-IoT or LTE-M), Power Saving Mode (PSM), and extended Discontinuous Reception (eDRX) modes and parameters.
```

Reply:
Added mention of GPS and the other suggestions. DId not include LTE preference as that's not supported in the link controller and not in the modem. Currently LTE mode preference is indirectly handled by `CONFIG_LTE_NETWORK_USE_FALLBACK`.